### PR TITLE
feat(Cda): presence rewrite

### LINKED ISSUE #5924: cda.pl | www.cda.pl
### Description

Presence is not showing

### Steps to Reproduce

1. Install presence
2. Play any video on cda.pl

### Screenshots

_No response_

### Expected Behavior

Showing video in presence when im watching any

### Actual Behavior

Presence is not showing

### Page URL

https://www.cda.pl/video/68324455c

### Operating System (OS)

Windows 10

### Browser

Chrome

### Browser Version

99.0.4844.51 64 bit

### LINKED ISSUE #6061: Cda | www.cda.pl
### Description

The premid on this page is not working, it is not communicating to discord as if it were not detecting cda

### Steps to Reproduce

1. Go to the link
2. Click something
3. Discord's activity isnt showing that 

### Screenshots

_No response_

### Expected Behavior

Discord doesnt showing the activity from this page

### Actual Behavior

Discord doesnt showing the activity from this page

### Page URL

https://www.cda.pl

### Operating System (OS)

windows 11

### Browser

opera

### Browser Version

LVL3 (core: 85.0.4341.65)

### PR DESCRIPTION
**Pull request by xKubuuus**
## Description 
cda.pl presence rewrite: fixed presence with features (buttons etc.)

Closes #6061 

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>
https://imgur.com/a/9vZ0cDz

</details>

### CODE REVIEW

File: websites/C/cda.pl/presence.ts [hunk 1]
```diff
@@ -11,71 +11,95 @@ const presence = new Presence({
 
 presence.on("UpdateData", async () => {
 	const presenceData: PresenceData = {
-			largeImageKey: "logo",
+			largeImageKey: "https://i.imgur.com/AD1naGM.png",
 			startTimestamp: browsingTimestamp
 		},
-		pathname: string = document.location.pathname.toLowerCase();
+		path = document.location.pathname.toLowerCase();
```
Comment (EncryptedDev) [inline websites/C/cda.pl/presence.ts:17]: Please keep the variable named `pathname` instead 
```suggestion
		pathname = document.location.pathname.toLowerCase();
```

**Comment by ImRodry** [comment]
@marzeq

File: websites/C/cda.pl/presence.ts [hunk 2]
```diff
@@ -11,71 +11,95 @@ const presence = new Presence({
...
-		) {
-			presenceData.details = `(Reklama) ${title}`;
+	if (path == "/") presenceData.details = "Przegląda stronę główną";
+	else if (path.endsWith("/ustawienia"))
+		presenceData.details = "Przegląda ustawienia";
+	else if (path.endsWith("/poczta")) presenceData.details = "Przegląda pocztę";
+	else if (path.endsWith("/premium"))
+		presenceData.details = "Przegląda filmy CDA Premium";
+	else if (path.endsWith("/powiazanekonta"))
+		presenceData.details = "Przeglądanie powiązanych kont";
```
Comment (marzeq) [inline websites/C/cda.pl/presence.ts:26]: This should be "Przegląda powiązane konta" to be consistent with the other presences.

Comment (marzeq) [inline websites/C/cda.pl/presence.ts:27]: This should be "Przegląda powiązane konta" to be consistent with the other presences.

File: websites/C/cda.pl/presence.ts [hunk 3]
```diff
@@ -11,71 +11,95 @@ const presence = new Presence({
...
+	else if (path.endsWith("/historia"))
+		presenceData.details = "Przeglądanie historii oglądania";
```
Comment (marzeq) [inline websites/C/cda.pl/presence.ts:29]: Same as previous one.

File: websites/C/cda.pl/presence.ts [hunk 4]
```diff
@@ -11,71 +11,95 @@ const presence = new Presence({
...
+	else if (path.endsWith("/obejrzyj-pozniej"))
+		presenceData.details = 'Przeglądanie "Obejrzyj później"';
```
Comment (marzeq) [inline websites/C/cda.pl/presence.ts:31]: Same here.

File: websites/C/cda.pl/presence.ts [hunk 5]
```diff
@@ -11,71 +11,95 @@ const presence = new Presence({
...
+			}`;
+			presenceData.state = `Teraz: ${
+				document.querySelector(
+					"#gora > div.tv-one-canal-list > div.tv-guide-well.tv-f > div.tv-list-pr.tv-list-pr-actual > span > span.tvm-b.relative > div.tvpr-title-header > h2"
+				).textContent
+			}`;
+		}
+	} else if (path.includes("/video")) {
+		if (path === "/video")
+			presenceData.details = "Przegląda wideo użytkowników";
```
Comment (marzeq) [inline websites/C/cda.pl/presence.ts:49]: In my opinion this should be changed to "Przegląda stronę głowną wideo", as `/video` with no query string is the de-facto home page for videos.

File: websites/C/cda.pl/presence.ts [hunk 6]
```diff
@@ -11,71 +11,95 @@ const presence = new Presence({
...
+		else {
+			presenceData.details = "Przegląda wideo użytkowników";
```
Comment (marzeq) [inline websites/C/cda.pl/presence.ts:51]: Why is this plural here?

File: websites/C/cda.pl/presence.ts [hunk 7]
```diff
@@ -11,71 +11,95 @@ const presence = new Presence({
...
-		} else {
+			presenceData.details = document.querySelector(
+				"#naglowek > span > span > h1"
+			).textContent;
+			presenceData.state = `Przesłał: ${
+				document.querySelector(
+					"#leftCol > div:nth-child(2) > div.DescrVID > div.DescrVID-left > div > div > div > div:nth-child(1) > a > span > span"
+				).textContent
+			}`;
+			presenceData.buttons = [{ label: "Przejdź do filmu", url: document.URL }];
```
Comment (marzeq) [inline websites/C/cda.pl/presence.ts:65]: A better fitting label would be "Obejrzyj film".

**Review by marzeq — CHANGES_REQUESTED** [review]

**Comment by marzeq** [comment]
Strings are all good now.

**Comment by ImRodry** [comment]
The commit history of this branch is a mess, I'll be squashing every commit in order to fix conflicts

**Review by EncryptedDev — DISMISSED** [review]

**Comment by ImRodry** [comment]
Please resolve the CI issues

File: websites/C/cda.pl/dist/metadata.json [hunk 8]
```diff
@@ -12,8 +12,8 @@
 		"vi_VN": "Dịch vụ cung cấp VOD độc lập lớn nhất Phần Lan. Cung cấp phim và sê-ri chất lượng cao."
 	},
 	"url": "www.cda.pl",
-	"version": "1.0.12",
-	"logo": "https://i.imgur.com/SQlMvJh.jpg",
+	"version": "2.0.0",
+	"logo": "https://i.imgur.com/AD1naGM.png",
```
Comment (ImRodry) [inline websites/C/cda.pl/dist/metadata.json:16]: don't know where you got this logo from but according to [this page](https://www.cda.pl/logotyp) the old one is correct

File: websites/C/cda.pl/presence.ts [hunk 9]
```diff
@@ -11,71 +11,94 @@ const presence = new Presence({
 
 presence.on("UpdateData", async () => {
 	const presenceData: PresenceData = {
-			largeImageKey: "logo",
+			largeImageKey: "https://i.imgur.com/AD1naGM.png",
```
Comment (ImRodry) [inline websites/C/cda.pl/presence.ts:14]: same as above, please keep the old logo

File: websites/C/cda.pl/presence.ts [hunk 10]
```diff
@@ -11,71 +11,94 @@ const presence = new Presence({
...
+			"#placeholder-foldery > div:nth-child(1) > div.panel-heading.smoke"
+		).textContent === "Foldery "
+	) {
+		presenceData.details = `Przegląda folder od ${
+			document.location.pathname.split("/")[1]
+		}`;
+		presenceData.state = document.querySelector(
+			"#folder-replace > div.panel-heading.smoke > span > span > a"
+		).textContent;
+		presenceData.buttons = [{ label: "Przejdź do folderu", url: document.URL }];
```
Comment (ImRodry) [inline websites/C/cda.pl/presence.ts:89]:
```suggestion
		presenceData.buttons = [{ label: "Przejdź do folderu", url: document.location.href }];
```

**Review by ImRodry — CHANGES_REQUESTED** [review]

File: websites/C/cda.pl/dist/metadata.json [hunk 11]
```diff
@@ -22,4 +22,4 @@
 		"movies",
 		"vod"
 	]
-}
\ No newline at end of file
+}
```
Comment (ImRodry) [inline websites/C/cda.pl/dist/metadata.json:25]: do me a favor and remove this line pls :P

**Review by ImRodry — APPROVED** [review]
Approving but pls address the review

**Review by EncryptedDev — APPROVED** [review]

**Comment by EncryptedDev** [comment]
Cheers for the PR 🎉

**Pull request merged**